src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -66,33 +68,51 @@ type FiatAccountFormSchema<T extends FiatAccountSchema> = {
     | ComputedParam<FiatAccountSchemas[T], Property>
 }
 
-const getAccountNumberSchema = (implicitParams: {
-  country: string
-  fiatAccountType: FiatAccountType
-}): FiatAccountFormSchema<FiatAccountSchema.AccountNumber> => ({
-  institutionName: {
-    name: 'institutionName',
-    label: i18n.t('fiatAccountSchema.institutionName.label'),
-    regex: /.+/,
-    placeholderText: i18n.t('fiatAccountSchema.institutionName.placeholderText'),
-    keyboardType: 'default',
-  },
-  accountNumber: {
-    name: 'accountNumber',
-    label: i18n.t('fiatAccountSchema.accountNumber.label'),
-    regex: /^[0-9]{10}$/,
-    placeholderText: i18n.t('fiatAccountSchema.accountNumber.placeholderText'),
-    errorMessage: i18n.t('fiatAccountSchema.accountNumber.errorMessage'),
-    keyboardType: 'number-pad',
+const getAccountNumberSchema = (
+  implicitParams: {
+    country: string
+    fiatAccountType: FiatAccountType
   },
-  country: { name: 'country', value: implicitParams.country },
-  fiatAccountType: { name: 'fiatAccountType', value: FiatAccountType.BankAccount },
-  accountName: {
-    name: 'accountName',
-    computeValue: ({ institutionName, accountNumber }) =>
-      `${institutionName} (${getObfuscatedAccountNumber(accountNumber!)})`,
-  },
-})
+  countryOverrides: FiatConnectSchemaCountryOverrides<FiatAccountSchema.AccountNumber>
+): FiatAccountFormSchema<FiatAccountSchema.AccountNumber> => {
+  // NOTE: the schema for overrides supports overriding any field's regex or
+  // errorMessage, but the below currently applies it to just accountNumber.
+  // This can be extended to support overriding other params and applied more
+  // generically if more fields/schemas require it.
+  const overrides = countryOverrides[implicitParams.country]
+
+  return {
+    institutionName: {
+      name: 'institutionName',
+      label: i18n.t('fiatAccountSchema.institutionName.label'),
+      regex: /.+/,
+      placeholderText: i18n.t('fiatAccountSchema.institutionName.placeholderText'),
+      keyboardType: 'default',
+    },
+    accountNumber: {
+      name: 'accountNumber',
+      label: i18n.t('fiatAccountSchema.accountNumber.label'),
+      regex: overrides?.accountNumber?.regex
+        ? new RegExp(overrides.accountNumber.regex)
+        : /^[0-9]+$/,
+      placeholderText: i18n.t('fiatAccountSchema.accountNumber.placeholderText'),
+      errorMessage: overrides?.accountNumber?.errorString
+        ? i18n.t(
+            `fiatAccountSchema.accountNumber.${overrides.accountNumber.errorString}`,
```

Comment:
I think you should check if `overrides.accountNumber.errorString === 'errorMessageDigitLength`. If you don't, old clients won't be able to handle new errorStrings from Remote Config

Reply:
we can push new translation strings OTA right? cc @kathaypacific

Reply:
Ah how could I forget OTA 🤦 !